lib/IDE/CodeCompletion.cpp

```diff
@@ -5943,10 +5943,13 @@ void CodeCompletionCallbacksImpl::doneParsing() {
       // module loading, for example, the module file is corrupted.
       if (!ModuleFilename.empty()) {
         auto &Ctx = TheModule->getASTContext();
-        CodeCompletionCache::Key K{ModuleFilename, TheModule->getName().str(),
-                                   AccessPath, Request.NeedLeadingDot,
-                                   SF.hasTestableImport(TheModule),
-                                   Ctx.LangOpts.CodeCompleteInitsInPostfixExpr};
+        CodeCompletionCache::Key K{
+            ModuleFilename,
+            TheModule->getName().str(),
+            AccessPath,
+            Request.NeedLeadingDot,
+            SF.hasTestableOrPrivateImport(AccessLevel::Internal, TheModule),
```

Comment:
This is NFC compared to before. This was only rewritten to use the SF.hasTestableOrPrivateImport instead of the hasTestableImport function. The outcome should be the same (whatever that is).

Reply:
Right, that's not the problem. What I'm worried about is that the private-imported members either won't show up in code completion, or they *will* and could incorrectly show up in the cache for that module.